switchmap/poller/snmp/async_snmp_manager.py

```
@@ -0,0 +1,949 @@
+                
+                #! adding more specific logs here
+                if error_indication:
+                    print(f"BULK error indication: {error_indication}")
+                    break
+                elif error_status:
+                    print(f" BULK error status: {error_status.prettyPrint()} at {error_index}")
+                    break
+                
+                #Check if we got any response
+                if not var_bind_table:
+                    print(f"No more data")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)